test/test_torch.py

```diff
@@ -8410,6 +8407,12 @@ def _test_view(self, cast):
         self.assertEqual(tensor.view(6, 2, 1), contig_tensor.view(6, 2, 1))
         self.assertEqual(tensor.view(1, 6, 2, 1), contig_tensor.view(1, 6, 2, 1))
 
+        # test viewing the same size
+        _tensor = tensor.view([3, 4])
```

Comment:
this isn't going to catch the issue, as I mentioned above, this has to be done in a non-variable scope.

Reply:
I see, should I still leave them here though